content/en/infrastructure/livecontainers/configuration.md

```diff
@@ -177,9 +177,8 @@ The following table presents the list of collected resources and the minimal Age
 | ServiceAccounts | 7.27.0 | 1.19.0 | 2.30.9 | 1.17.0 |
 | Services | 7.27.0 | 1.11.0 | 2.10.0 | 1.17.0 |
 | Statefulsets | 7.27.0 | 1.15.0 | 2.20.1 | 1.16.0 |
-| VerticalPodAutoscalers | 7.27.0 | 7.43.0 | 3.6.8 | 1.25.0 |
```

Comment:
@xlucas We sometimes document beta information. Do you want to maybe leave this in with a "(beta)" note on it

Reply:
@kayayarai in that case we'll just want to remove it since it's not even available to customers yet